packages/express/src/middleware/index.ts

```diff
@@ -80,3 +83,16 @@ export function expressErrorHandler(appsignal: Client): ErrorRequestHandler {
     return next(err)
   }
 }
+
+function filterHeaders(headers: IncomingHttpHeaders): HashMap<any> {
+  let filtered: HashMap<any> = {}
+  const headersAllowList = global.__APPSIGNAL__.config.data.requestHeaders || []
```

Comment:
Let's use the client's config object to fetch the request headers for now. Until we have an accessor for the client/config on on the Appsignal module. That way we hopefully don't need to add the global type definition in this package as well.

We should have the client available here, because it's given to the `expressMiddleware` function.

Reply:
Yup, noted that in the commit.

Reply:
Yes, but let's not access it directly on the global object here until we have an accessor. We have the client object here already, we don't need to fetch it from the global object.